docs/data/tree-view/rich-tree-view/editing/EditLeaves.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { TreeViewBaseItem } from '@mui/x-tree-view/models';
+import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
+import { TreeItem2 } from '@mui/x-tree-view/TreeItem2';
+import { useTreeViewApiRef } from '@mui/x-tree-view/hooks';
+
+type ExtendedTreeItemProps = {
+  editable?: boolean;
+  id: string;
+  label: string;
+};
+
+const MUI_X_PRODUCTS: TreeViewBaseItem<ExtendedTreeItemProps>[] = [
+  {
```

Comment:
I think we can use the same dataset everywhere, the one with charts and tree view.
And I unified the CSS of the `Box` / `Stack` used in all the demos, I let you check if yours is aligned :+1: